src/material-examples/tree-checklist/tree-checklist-example.ts

```diff
@@ -168,10 +168,13 @@ export class TreeChecklistExample {
     return flatNode;
   }
 
-  /** Whether all the descendants of the node are selected */
+  /** Whether all the descendants of the node are selected. */
   descendantsAllSelected(node: TodoItemFlatNode): boolean {
     const descendants = this.treeControl.getDescendants(node);
-    return descendants.every(child => this.checklistSelection.isSelected(child));
+    const descAllSelected = descendants.every(child =>
```

Comment:
nit: you can just use `return descendants.every(...)` here instead of assigning it to a constant.